.github/actions/setup-python/action.yml

```diff
@@ -40,6 +40,13 @@ runs:
         path: .venv
         key: "venv-${{ runner.os }}-${{ runner.environment }}-${{ hashFiles('requirements**.lock') }}"
 
+    - name: Export PYO3 Path
+      shell: bash
+      # Figure out a way to dynamically get python version
+      run: |
+        echo "PYO3_PYTHON=/home/runner/work/vortex/vortex/.venv/bin/python" >> $GITHUB_ENV
```

Comment:
wait, is this going to work on self-hosted runners @AdamGS ?

Reply:
I think it should since they still run in containers and this is the container relative path

Reply:
yeah, benchmarks run just fine